libcontainer/container_linux.go

```diff
@@ -2321,6 +2321,18 @@ func (c *Container) bootstrapData(cloneFlags uintptr, nsMaps map[configs.Namespa
 		})
 	}
 
+	// write boottime and monotonic time ns offsets.
+	if c.config.TimeOffsets != nil {
```

Comment:
I think the tests are failing because this can now be non-`nil` after the specconv change. This should be fixed with:

```suggestion
	if config.Namespaces.Contains(configs.NEWTIME) && c.config.TimeOffsets != nil {
```

Reply:
I'll recheck the reason for failure